src/slacker/client/cluster.clj

```diff
@@ -82,43 +82,49 @@
 (defn ^:no-doc group-call-results [grouping-results
                                    grouping-exceptions
                                    call-results]
-  (let [req-info (dissoc (first call-results) :result :cause)]
-    (doseq [r (filter :cause call-results)]
-      (logging/info (str "error calling "
+  (let [req-info (dissoc (first call-results) :result :cause)
+        failed-results (filter :cause call-results)]
+    (doseq [r failed-results]
+      (logging/warn (str "error calling "
```

Comment:
maybe warn is more appropriate ? there's something wrong with the rpc after all.